pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tesla"
-version = "2.3.1"
```

Comment:
Don't bump the component version. That's done by CI.

Reply:
Updated! Sorry about that.